src/AnimalClub/Animals/Examples/Goat.hs

```diff
@@ -125,3 +126,8 @@ rotateArbTrans θ = ArbTrans (conv_Rotation_M44 (axisAngle (V3 0 1 0) θ) !*!)
 octopusWorm :: AnimalNode Float
 octopusWorm = asPhantom $ mans "root" (relV3 0 0 0) (Abs 0.2) $
   map (\r -> flipAnimalNode (rotateArbTrans (r*2*pi/8)) (defTransFlag Same) flipWorm) [0..7]
+
+
+-- | Animalnode representation of canonical cube
+cube :: AnimalNode Float
```

Comment:
Cube animal is best animal

Reply:
I got the diameter wrong so it's actually half cube ._.